### PR TITLE
Added "PicoScope 7 T&M.app" v7.1.2

### PR DESCRIPTION
**Pull request by HendrikLamar**
So far, only the PicoScope Beta software was available. Now, I added the stable version of the software in it's most recent version.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).  
  - In my opinion this is misleading in since nobody knows what the "T & M" stands for and might confuse people.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/picoscope.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "picoscope" do
+  version "7.1.2.15463"
+  sha256 "2c0dbeb737ee0f94eb4dad296da9de808122f468bf2410fceab466a7aa80a34d"
+
+  url "https://www.picotech.com/download/software/sr/PicoScope_#{version}_TnM_Stable.pkg"
+  name "PicoScope"
+  desc "Test & Measurement Software for Picotech Oscilloscopes"
```
Comment (razvanazamfirei) [inline Casks/picoscope.rb:7]:
```suggestion
  desc "Oscilloscope test and measurement software"
```

File: Casks/picoscope.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  homepage "https://www.picotech.com/"
+
+  livecheck do
+    url "https://www.picotech.com/downloads/_lightbox/picoscope-7-stable-for-macos"
+    regex(%r{href=.*?/PicoScope_(\d+(?:.\d+)*).*\.pkg}i)
```
Comment (razvanazamfirei) [inline Casks/picoscope.rb:12]:
```suggestion
    regex(%r{href=.*?/PicoScope_(\d+(?:.\d+)+).*?Stable\.pkg}i)
```

File: Casks/picoscope.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+  #  app "PicoScope 7 T&M.app" -> "Error: It seems your app is not found in
+  #  /opt/homebrew/Casksroom/..."
```
Comment (razvanazamfirei) [inline Casks/picoscope.rb:16]: Would you mind clarifying what this means?

**Review by razvanazamfirei — DISMISSED** [review]
Hi @HendrikLamar, thanks so much for opening this! Some comments below!

**Comment by miccal** [comment]
Thank you @HendrikLamar.

**Review by miccal — APPROVED** [review]

File: Casks/picoscope.rb (hunk 3 above)
Comment (HendrikLamar) [reply Casks/picoscope.rb]: Sure.
According to the docs the `app` stanza is kind of needed. However, the given stanza ends up in the error message

``` bash
Error: It seems the App source '/opt/homebrew/Caskroom/picoscope/7.1.2.15463/PicoScope 7 T&M.app' is not there.
```

when `brew install picoscope` is exectued.

However, if you are all fine without the `app` stanza then we can delete the lines 15 & 16.

**Pull request merged**